boilerplate/openshift/osd-golang-operator-osde2e/e2e-harness-generate.sh

```diff
@@ -6,14 +6,14 @@ cmd=${0##*/}
 
 usage() {
     cat <<EOF
-Usage: $0 OPERATOR_NAME CONVENTION_DIR
+Usage: $0 OPERATOR_NAME OSDE2E_CONVENTION_DIR
  
 EOF
     exit -1
 }
 
 OPERATOR_NAME=$1
-CONVENTION_DIR=$2
+OSDE2E_CONVENTION_DIR=$2
```

Comment:
nit: since this is a shell script, and it's taking this as an argument rather than from the env, it wasn't necessary to rename this instance of the variable.